packages/rum-recorder/src/domain/rrweb/types.ts

```diff
@@ -147,18 +147,20 @@ export interface MousePosition {
   timeOffset: number
 }
 
-export enum MouseInteractions {
-  MouseUp = 0,
-  MouseDown = 1,
-  Click = 2,
-  ContextMenu = 3,
-  DblClick = 4,
-  Focus = 5,
-  Blur = 6,
-  TouchStart = 7,
-  TouchEnd = 9,
+export const MouseInteractions = {
+  MouseUp: 0 as const,
+  MouseDown: 1 as const,
+  Click: 2 as const,
+  ContextMenu: 3 as const,
+  DblClick: 4 as const,
+  Focus: 5 as const,
+  Blur: 6 as const,
+  TouchStart: 7 as const,
+  TouchEnd: 9 as const,
```

Comment:
FMU, it would behave the same with
```ts
export const xxx = {
  foo: 1,
  ...
} as const
```

and it could remove some `as const` noise

Reply:
Nice!